securedrop/journalist_app/api2/__init__.py

```diff
@@ -116,22 +136,22 @@ def data() -> Response:
         for event in sorted(events, key=lambda e: int(e.id)):
             result = handler.process(event)
```

Comment:
Event handlers will need to be passed `minor` so that they can version-check consistently:

```patch
-        current_version = json_version(source.to_api_v2())
+        current_version = json_version(source.to_api_v2(minor))
```

Reply:
maybe we have it as a member variable on handler and turn the event handlers into normal methods instead of static ones?